compress.bash

```diff
@@ -11,7 +11,7 @@
 # Name of the ROM. No Spaces Please.
 # Example: CyanogenMod
 
-name=Cyanogenmod
+name=Cyanogenmod14.0
```

Comment:
bro..please unblock me..trust me i wont annoy you agian..ever
today is the first and last time...trust me
A sincere request, :)